maquetado/index.html

```diff
@@ -8,6 +8,21 @@
  <link rel="stylesheet" type="text/css" href="scss/out/application.css">
 </head>
 <body>
-  <h1 class="title">Hello world</h1>      
+  <div>
+    <p class="text-big text-bold book-element">Título del libro</p>
+    <p class="text-grey book-element">(género)</p>
+  </div>
+  <div>
+    <p class="text-bold book-element">Autor del libro:</p>
+    <p class="text-grey book-element">Nombre del autor del libro</p>
+  </div>
+  <div>
+    <p class="text-bold book-element">Editorial:</p>
+    <p class="text-grey book-element">Nombre de la editorial</p>
+  </div>
+  <div>
+    <p class="text-bold book-element">Año de publicación:</p>
+    <p class="text-grey book-element">Año de publicación</p>
+  </div>
```

Comment:
Although I get why you need these divs (to have block elements), we try not to have plain div tags (without any attributes like classes) 'cause they don't tell us much and most of the times you can achieve the same through CSS. 

I am not against using inline as display, I like that you thought of this but it's not very semantic. The idea is to have a structure that it's actually easy to understand rather than one that gets the job done. With that in mind, you could do this through `flex`, making your divs `book-property` and applying flex to them.